src/components/Auth/parseErrorMessages.js

```diff
@@ -1,5 +1,5 @@
 const parseErrorMessages = (error) => {
-  let heading = 'Could not log in';
+  let heading = 'Could not login';
```

Comment:
I think it should be `log in` still, like verb vs noun thing : https://www.grammar.com/log_in_vs._login

Reply:
ping @marians